src/main/kotlin/carRacing/CarRacingManager.kt

```diff
@@ -0,0 +1,33 @@
+package carRacing
+
+abstract class CarRacingManager {
```

Comment:
생성시점부터 완전한  cars List를 받으면 어떨까요?

```suggestion
abstract class CarRacingManager(private val cars: List<Car> = listOf()) {
```